src/lavfmuxer.cpp

```diff
@@ -61,6 +61,15 @@ lavfmuxer::lavfmuxer(const char *format, uint32_t audiostreammask, mpgfile &mpg,
   s->sample_aspect_ratio = codec->sample_aspect_ratio;
   s->time_base = codec->time_base;
 
+#if LIBAVCODEC_VERSION_INT >= AV_VERSION_INT(57, 14, 0)
+  // Use Parameters instead of Codec, necessary to get correct results, prevent crash and correct display of the output of av_dump
+  int ret = avcodec_parameters_from_context(s->codecpar, codec);
```

Comment:
Just a nitpic, maybe we can move the declaration of ret outside the `#if`?

Reply:
Indeed you are right. I would propose to do the int ret=0; directly before. As I am new, shall I provide a new patch or will you just consider it in by the merge?

Reply:
Like you want - either change that commit and force push it to your branch, or I can do it.

Reply:
Ok, I changed both on my branch. New commit is ba1a7620dd220573e204a02cc3b3277c7a2dee92.